projects/dotcms-field-elements/src/components/dot-input-calendar/dot-input-calendar.e2e.ts

```diff
@@ -74,38 +69,32 @@ describe('dot-input-calendar', () => {
                 await page.waitForChanges();
                 expect(input.getAttribute('disabled')).toBeNull();
             });
+        });
 
-            it('should render disabled attribute with invalid value', async () => {
-                element.setProperty('disabled', { test: 'test' });
+        describe('min', () => {
+            it('should not render  attribute by default', async () => {
                 await page.waitForChanges();
```

Comment:
if you're not changing anything I think this is not needed

Reply:
is just to make sure that the default value is empty.